templates/html/templates/common/macros.html

```diff
@@ -99,9 +109,9 @@
 {%- endmacro %}
 
 
-{% macro render_description_item_local_iri(result_set, property_id, property_label, link_label_property_id=None, accordion_style=False) -%}
+{% macro render_description_item_iri(result_set, property_id, property_label, rdfa_property, link_label_property_id=None, accordion_style=False, accordion_content_state="hide", external_link=False) -%}
     {% if conf.debug %}
-        <p class="debug-render">render_description_item_local_iri: {{ property_id }}</p>
+        <p class="debug-render">render_description_item_iri: {{ property_id }}</p>
```

Comment:
so now we have either a `div` or a `dl`; will this be an issue in writing the tests?